ansible_files/todo-playbook.yml

```diff
@@ -0,0 +1,69 @@
+- name: Run ToDo app on webserver
+  hosts: webservers
+  remote_user: ec2-user
+  vars_prompt:
+    - name: trello_key
+      prompt: What is your trello key?
+      private: no
+      default: '4effc87f8e3d5da536c3f923d87aacd6'
+
+    - name: trello_token
+      prompt: What is your trello token?
+      private: no
+      default: 'fe1682e1d7089060fdbd4321a87c27b598eb381c348707a7b23a345c5405af31'
```

Comment:
Don't commit real credentials to GitHub! You should at least remove the default value for trello_token and trello_key.

If you care about your Trello account, you ought to rotate your credentials.